extensions/wikia/Blogs/js/Blogs.js

```diff
@@ -146,7 +146,7 @@ Blogs.save = function( event ) {
 	if (oForm.length > 0) {
 		document.body.style.cursor = "wait";
 		var textfield = $( "#blog-comm-textfield-" + id );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the declaration of the unused 'textfield' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)

[project-id]: wikia:app
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "textfield" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

[project-id]: wikia:app